src/server/api/routers/rss.ts

```diff
@@ -159,7 +159,9 @@ const processItemContent = (content: string, dangerousAllowSanitizedItemContent:
     });
   }
 
-  return encode(content);
+  return encode(content, {
+    mode: 'nonAsciiPrintableOnly',
```

Comment:
Won't this no longer encode < and >? 

from [npm](https://www.npmjs.com/package/html-entities#usage)
```
encode('< > " \' & ©', {mode: 'nonAsciiPrintableOnly', level: 'xml'});
// -> '< > " \' & &#169;'
```

Reply:
The level is only for the `&#169;` i guess

Reply:
Updated 👍